web/pdf_scripting_manager.js

```diff
@@ -305,6 +305,27 @@ class PDFScriptingManager {
           }
           this._pdfViewer.currentScaleValue = value;
           break;
+        case "SaveAs":
+          this._eventBus.dispatch("save", { source: this });
+          break;
+        case "FirstPage":
+          this._pdfViewer.currentPageNumber = 1;
+          break;
+        case "LastPage":
+          this._pdfViewer.currentPageNumber = this._pdfViewer.pagesCount;
+          break;
+        case "NextPage":
+          this._pdfViewer.nextPage();
+          break;
+        case "PrevPage":
+          this._pdfViewer.previousPage();
+          break;
+        case "ZoomViewIn":
```

Comment:
Similar to the "zoom"-case in the existing code, please add the following check to prevent issues:
```javascript
if (isInPresentationMode) {
  return;
}
```